app/models/application_record.rb

```diff
@@ -69,7 +69,7 @@ def self.aggregate_of(association)
     class_eval do
       define_method(:rebuild!) do |children|
         transaction do
-          self.send(association).all_except(children).delete_all
+          self.send(association).all_except(children).destroy_all
```

Comment:
:warning: Although using `destroy_all` is necessary for running the `destroy`  hook, this has a performance impact, since instead of running just a simple `delete from .. where ...` you will run a `select`  and then a single delete for each of the found records.

Reply:
I agree with this one, but it stills improves the performance and allows us to use it in #61.
It is not that frequent to remove several children content at once, and even if we do so, it wouldn't be such a terrible workload.